openadapt/record.py

```diff
@@ -134,7 +134,6 @@ def process_events(
         terminate_event: An event to signal the termination of the process.
     """
 
-    utils.configure_logging(logger, LOG_LEVEL)
```

Comment:
Thanks @nweston !

Can you please clarify why it was necessary to remove these calls to `utils.configure_logging`?

Reply:
It's not strictly necessary -- adding the lock is sufficient to fix the bug. But these calls appear to be redundant (`utils.configure_logging` is already called at module level with the same arguments), and it's possible that there could be other race conditions, e.g. if one thread is outputting log messages while another is configuring. So removing them seemed like the safest option.

But if there's a reason to keep them, we can do that and the duplicate log messages will still be fixed.